### DISCUSSION TITLE
This user account is blocked.

### DISCUSSION THREAD

**Discussion by groomcars** [vercel/community#5377] [Account Issue]
### Question

I have registered an Account using Github and when I try to login using Github or email getting error as below:
<img width="1440" alt="image" src="https://github.com/vercel/community/assets/155832562/ea3beb63-541c-4ea6-b125-44b9cb12b4ad">

**Comment by Yos666666** [comment]
Good luck, I've been waiting for a response from support for this problem for over 3 weeks.
